src/nvim/testdir/test_quickfix.vim

```diff
@@ -5087,4 +5087,70 @@ func Test_qftextfunc()
   call Xtest_qftextfunc('l')
 endfunc
 
+func Qftextfunc_other1(d)
+  return map(getloclist(d.winid, {'id' : d.id,
+    \ 'items' : 1}).items[d.start_idx-1:d.end_idx-1],
+    \ "'Line ' .. v:val.lnum .. ', Col ' .. v:val.col")
+endfunc
+
+func Qftextfunc_other2(d)
+  return map(getloclist(d.winid, {'id' : d.id,
+    \ 'items' : 1}).items[d.start_idx-1:d.end_idx-1],
+    \ "'Ligne ' .. v:val.lnum .. ', Colonne ' .. v:val.col")
+endfunc
+
+" Test for updating a location list for some other window and check that
+" 'qftextfunc' uses the correct location list.
+func Test_qftextfunc_other_loclist()
+  %bw!
+  call setloclist(0, [], 'f')
+
+  " create a window and a location list for it and open the location list
+  " window
+  lexpr ['F1:10:12:one', 'F1:20:14:two']
+  let w1_id = win_getid()
+  call setloclist(0, [], ' ',
+        \ {'lines': ['F1:10:12:one', 'F1:20:14:two'],
+        \  'quickfixtextfunc':
+        \    Qftextfunc_other1})
```

Comment:
Don't delete the original lines; comment them out instead.

Reply:
Sorry, I almost missed this, but `quickfixtextfunc` takes a string value of the function's name.

```vim
        \  'quickfixtextfunc':
        \    'Qftextfunc_other2'})
```